R/utilities.R

```diff
@@ -819,3 +819,97 @@ hist_config_update <- function(df){
 summarize_remote_data <- function(){
   system("ssh ucla '/srv/shiny-server/scraper_utilities/summarize.R'")
 }
+
+## calculate latest facility-level and state increases
+
+#' @param scrape_df facility-level data to use
+#' @param outbreaks_sheet_loc optional, google sheets ID to use if writing to google sheets
+#' @param metric character string of the metric to calc 
+#' @param delta_days integer, number of days to calculate increase over  
+#' @param num_fac integer, number of facilities to plot 
+#' @param overwrite_data T/F, write output data to google sheet specified in outbreaks_sheet_loc
+#' 
+#' @return dataframe with highest state and facility-level outbreaks
+
+## code borrowed from: https://github.com/uclalawcovid19behindbars/behindbarstools/blob/master/R/plot_recent_fac_increases.R
+track_recent_covid_increases <- function(
+  scrape_df, 
+  outbreaks_sheet_loc,
```

Comment:
Can we have a default for teh parameter outbreaks_sheet_loc? Would be helpful to know where we are expecting to write this directly in teh function